src/components/RightSidebar/RightSidebar.vue

```diff
@@ -254,6 +287,30 @@ export default {
 			emit('show-settings')
 		},
 
+		async handleUpdateDescription(description) {
+			this.isDescriptionLoading = true
+			const response = await setConversationDescription(this.token, description)
+			console.debug('response', response)
+			if (response) {
+				// If successful, immediately update the description in the store
+				this.$store.dispatch('setConversationDescription', {
```

Comment:
move the xhr request to the store: the store action should do the request and then commit the change

See https://github.com/nextcloud/spreed/blob/master/src/store/conversationsStore.js#L251 and maybe other properties we set the same way. This helps move some of the complexity to the store action instead of keeping it in the Vue components.

Reply:
ok, will give this a try!